Backend/app/routes/brand_dashboard.py

```
@@ -0,0 +1,1132 @@
+        if result.data:
+            for match in result.data:
+                # Get creator details
+                creator_result = supabase.table("users").select("*").eq("id", match["creator_id"]).execute()
+                if creator_result.data:
+                    creator = creator_result.data[0]
+                    match["creator_name"] = creator.get("username", "Unknown")
+                    match["creator_role"] = creator.get("role", "creator")
+                
+                matches.append(CreatorMatchResponse(**match))
+        
+        return matches
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)